specification/e-referrals-service-api.yaml

```diff
@@ -496,7 +496,44 @@ paths:
           $ref: 'components/schemas/responses/Forbidden.yaml'
         '500':
           $ref: 'components/schemas/responses/InternalServerError.yaml'
+  /R4/v1/PractitionerRole:
+    get:
+      security:
+        - bearerAuth: []
+      description: |
+        ## Overview
+        
+        Use this endpoint to retrieve a list of available e-RS roles (i.e. business functions) at each organisation where the user works.
+
+        ## Pre-requisites
+
+        In order to use this endpoint you must be an authenticated user
+
+        ## Use case
+
+        As an authenticated user
+
+        I need to retrieve a list of authorised e-RS roles for organisations I work for
 
+        So I can choose the appropriate role and organisation to work under when performing tasks.
+
+        ## Related endpoints
+
+        You can then use the e-RS roles to perform specific tasks using other e-RS endpoints.
+
+      summary: A028 - Retrieve e-RS business functions
```

Comment:
Does https://nhsd-jira.digital.nhs.uk/browse/ERSSUP-54610 need updating, now the "A" number is known?

Reply:
the number isn't known yet, I've just used the next one in sequence and we might have to update it once we get confirmation from NHS